inc/apiclient.class.php

```diff
@@ -274,7 +296,7 @@ public function setTicketTypeMapping()
     {
         $field_id = $_SESSION['metabase']['fields']['glpi_tickets.type'];
         $this->setFieldCustomMapping($field_id, __('Type'));
-        $data = $this->httpQuery("/api/field/$field_id/values", [
+        $data = $this->httpQuery("field/$field_id/values", [
```

Comment:
Why remove the `/api/` suffix?

Reply:
Because it is already defined in the getAPIBaseUri method.